pkgs/development/libraries/nss/default.nix

```diff
@@ -1,9 +1,14 @@
-{ stdenv, fetchurl, nspr, perl, zlib, sqlite, darwin, fixDarwinDylibNames, buildPackages, ninja
+{ stdenv, fetchurl, nspr, perl, zlib
+, sqlite, ninja
+, darwin, fixDarwinDylibNames, buildPackages
+, useP11kit ? true, p11-kit ? null
```

Comment:
```suggestion
, useP11kit ? true, p11-kit
```

Reply:
Is there an RFC or docs that recommends against asserts for optional dependencies?

Reply:
there is no documented guideline, however, in some cases they have been removed because they broke overriding.
b6187aa5967e4ed33809d2916ef9398d4a9d6af9 #73102

Reply:
Thank you.